config.yaml

```diff
@@ -0,0 +1,9 @@
+# Copyright 2024 Canonical Ltd.
+# See LICENSE file for licensing details.
+
+options:
+  log-level:
```

Comment:
**question** is this used anywhere? I checked the process, but it does not seem to take the log-level as an argument